CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## 7.5.5
+
+### Added
```

Comment:
Super nitpick, I'd put this under changed instead of added

Reply:
awesome thanks Mathias :D 
Fixed